<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IR Maker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>irmaker.png</normaloff>irmaker.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="spectral_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>1030</width>
      <height>330</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="spectrogram_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>1030</width>
      <height>330</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="sweep_box">
       <property name="title">
        <string>Sweep</string>
       </property>
       <widget class="QPushButton" name="browsesweep_button">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>250</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Browse sweep file</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="beg_freq">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>85</y>
          <width>89</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>20</string>
        </property>
        <property name="placeholderText">
         <string>in Hz</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="end_freq">
        <property name="geometry">
         <rect>
          <x>153</x>
          <y>110</y>
          <width>89</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>20000</string>
        </property>
        <property name="placeholderText">
         <string>in Hz</string>
        </property>
       </widget>
       <widget class="QLabel" name="begfreq_label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>87</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Beginning freq. (Hz)</string>
        </property>
       </widget>
       <widget class="QLabel" name="endfreq_label">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>112</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ending freq. (Hz)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sweepgen_button">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>150</y>
          <width>229</width>
          <height>61</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Sweep generator</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="response_box">
       <property name="title">
        <string>Response</string>
       </property>
       <widget class="QPushButton" name="loadfolder_button">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>40</y>
          <width>251</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Select recording(s) folder</string>
        </property>
       </widget>
       <widget class="QListView" name="files_list">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>75</y>
          <width>251</width>
          <height>136</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="output_box">
       <property name="title">
        <string>Output</string>
       </property>
       <widget class="QCheckBox" name="mpt_checkbox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>MP Transform</string>
        </property>
       </widget>
       <widget class="QComboBox" name="bitdepth_combo">
        <property name="geometry">
         <rect>
          <x>139</x>
          <y>165</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>16 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>32 bit</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="bitdepth_label">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>166</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bit depth</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="autosave_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>25</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Automatic output (creates &quot;IR&quot;
folder in recordings folder)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="customsave_radio">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Custom output</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseout_button">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>97</y>
          <width>151</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Browse output</string>
        </property>
       </widget>
       <widget class="QLabel" name="srate_label">
        <property name="geometry">
         <rect>
          <x>28</x>
          <y>135</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sample rate (Hz)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="srate">
        <property name="geometry">
         <rect>
          <x>138</x>
          <y>135</y>
          <width>106</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>in Hz</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="PlotWidget" name="ir_plot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1030</width>
      <height>315</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createir_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Create IR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playir_button">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>128</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Play IR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="about_button">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>5</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>About</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
